### ISSUE TITLE
Node.js Foundation User Feedback Meeting 2018-06-08: Tooling Feedback Session

### ISSUE THREAD

**Issue by boneskull** [nodejs/user-feedback#68]
# Time

**Friday, June 8th 2018**, 16:00 UTC (4:00 PM), or in [your local time](https://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+User%20Feedback+Meeting+2018-06-08&iso=20180608T16)

# Agenda

We will **gather user feedback** from Tooling group.  For this session, I've selected a few "hot-button" issues from #59 which we will discuss:

- Cross-platform problems (`fs`, `child_process`)
- `fs` is missing some key operations (`rimraf`, `mkdirp`, `fs-extra`, etc.)
- Node.js startup time

Otherwise, *anything goes* along these topics:

- What's great about writing CLI tools in Node.js?
- What could be better about writing CLI tools in Node.js?
- What upcoming features will have an impact on your tooling work?

We'll try to give everyone a chance to be heard, but time is limited!    Discussion points may also be added to #59, if you can't make the meeting.

# Invited

- Anyone working in Node.js CLI tooling who wants to provide feedback
- Anyone who attended the last Tooling session
- User feedback team members (@nodejs/user-feedback)

# Joining the meeting

- Zoom: https://zoom.us/j/168395218
- YouTube Live: https://youtube.com/c/nodejs+foundation/live 

# Definitions

For our purposes, **Tooling** can be considered "command-line programs written in Node.js".  Examples include tools which:

- build websites (e.g. Webpack, browserify)
- compile to/from JS (e.g. Babel, TypeScript)
- run test or static code checks (e.g. Mocha, ESLint)
- scaffold stuff (e.g. EmberCLI, create-react-app, Yeoman)
- run tasks (e.g. Grunt, Gulp)
- manage packages (e.g. npm, yarn)

and include:

- plugin ecosystem around any of the above
- libraries often consumed by CLI programs (e.g. `chalk`, `yargs`)

**Comment by boneskull** [comment]
@dshaw or @bnb Could you please provide video links?

**Comment by boneskull** [comment]
@dshaw, @bnb, @mhdawson Is it possible to have someone record minutes or take notes?

**Comment by mhdawson** [comment]
Link for the zoom meeting: https://zoom.us/j/168395218

**Comment by mhdawson** [comment]
In terms of minutes, we can create a google doc.  It works best to have a few people helping to capture what is being discussed, we'll also record the session as well.

**Comment by boneskull** [comment]
@mhdawson @dshaw @bnb Do we have a YouTube link?

**Comment by mhdawson** [comment]
Just a sec, pulling the standard link from another issue

**Comment by mhdawson** [comment]
We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at https://www.youtube.com/c/nodejs+foundation/live when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.

**Comment by bnb** [comment]
Did the meeting die? Looks like we may have a conflict with Mentorship :/

**Comment by dshaw** [comment]
@bnb No mentorship meeting at this time. Folks are logged on.

**Comment by dshaw** [comment]
Might be your local calendar.

**Comment by bnb** [comment]
@dshaw can you send the link again? It's telling me I need to be signed in to start the meeting 🤔

**Comment by dshaw** [comment]
https://zoom.us/j/168395218

**Comment by mhdawson** [comment]
Should be starting in around 3-5 minutes.

**Comment by bnb** [comment]
Not working :/
![image](https://user-images.githubusercontent.com/502396/41168655-97884f86-6b14-11e8-9082-b0ed04b54b19.png)

**Comment by dshaw** [comment]
@mhdawson Suggests logging out

**Comment by dshaw** [comment]
I keep getting logged out. @bnb log in on personal account.
